webcompat/templates/contributors/sub-nav.html

```diff
@@ -17,8 +17,8 @@
       </li>
 
       <li class="nested-sub-nav-item {% if request.url_rule.endpoint == 'contributors_bug_reproduce' %} sub-nav-active{% endif %}">
-        <a class="link-text" href="/contributors/reproduce-bug">
-          Reproduce a Bug
+        <a class="link-text" href="/contributors/triage-bug">
```

Comment:
```suggestion
        <a class="link-text" href="/contributors/reproduce-bug">
```

if the link name is changed, it will probably break.